pkg/system/phase4_configuring.go

```diff
@@ -60,6 +60,16 @@ func (r *Reconciler) ReconcilePhaseConfiguring() error {
 		}
 	}
 
+	// update noobaa-core version in reconciler struct
+	r.NBClient.SetAuthToken(r.SecretOp.StringData["auth_token"])
+	systemInfo, err := r.NBClient.ReadSystemAPI()
```

Comment:
Small thing - I would move this code into a small function and name it `ReconcileReadSystem()` and keep the result on the reconciler as `r.SystemInfo` as we will probably want it soon for more tasks in configuring the system. You still want to also set `r.CoreVersion = r.SystemInfo.Version` for the template.